### PR TITLE
Small fixes to the Instancing doc

### PR DESCRIPTION
**Pull request by noideatbh**
A few text changes and an updated image.

### CODE REVIEW

File: learning/step_by_step/instancing.rst [hunk 1]
```diff
@@ -97,7 +97,7 @@ Select one of the many copies of the balls and go to the property
 editor. Let's make it bounce a lot more, so look for the Bounce
 parameter and set it to 1.0:
 
-.. image:: /img/instedit.png
+.. image:: /img/instedit.PNG
```
Comment (akien-mga) [inline learning/step_by_step/instancing.rst]: I would prefer that you keep the original case.

**Comment by noideatbh** [comment]
Like that?

File: learning/step_by_step/instancing.rst (hunk 1 above)
Comment (noideatbh) [inline learning/step_by_step/instancing.rst]: Fixed.

**Comment by akien-mga** [comment]
Thanks. I'll use the "squash and merge" feature as 7 commits made with the GitHub editor is a bit overkill for the final amount of changes :)

For your next PR make sure to let the editor create a new branch, as your master branch will thus diverge from that of the main repo.

**Pull request merged**